lib/pipes/validators/validation.pipe.ts

```diff
@@ -8,6 +8,13 @@ export interface ValidationPipeOptions extends ValidatorOptions {
   transformOptions?: ClassTransformOptions;
 }
 
+/**
+ * Validates request bodies and gets them as DTOs.
```

Comment:
I think `request bodies` should be replaced with `request body values`, since the body is one thing with values inside. The same change should be made in the readme as well.